src/selenium/eventPage.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const EventPage = Object.create(BasePage);
+
+EventPage.getEventName = function() {
+  return this.find(By.css('h1')).getText().then(function(name) {
+    return name;
+  });
+};
+
+EventPage.checkTweetSection = function() {
+  // This will return an error if the tweet section is not there
+  return this.find(By.id('tweet'));
+};
+
+EventPage.getNavbarFooterBrokenLinks = function() {
+  // Both the promises return an array of links contained inside the container
+  const navbarPromise = this.getAllLinks(By.className('navbar-default'));
+  const footerPromise = this.getAllLinks(By.className('footer-container'));
+
+  const promiseArr = [];
+
+  promiseArr.push(navbarPromise);
+  promiseArr.push(footerPromise);
+
+  // Merges two arrays into one and remove duplicate elements
+  function mergeUniqueArr(arr1, arr2) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines. (padded-blocks)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)